### PR TITLE
Added pages of Login and Registration

### PR DESCRIPTION
**Pull request by Reinmir**
Added pages of Login and Registration. added some forms for this pages.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +12,10 @@
     "@types/react-dom": "^18.0.6",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router": "^6.4.1",
     "react-router-dom": "^6.4.1",
     "react-scripts": "5.0.1",
+    "sass": "^1.55.0",
```
Comment (IgorKravtsov) [inline package.json]: You need node-sass lib, not sass

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
-<Router>
-  <Routes>
-    <Route path='/login' element={<Login/>}></Route>
-    <Route path='/register' element={<Registration/>}></Route>
-  </Routes>
-</Router>
-
-</>
+    <>
+      <Router>
```
Comment (IgorKravtsov) [inline src/components/App/App.tsx]: It is better to move this <Router> wrapper outside the App.tsx component

File: src/components/FormBuilder/FormBuilder.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+interface FormBuilderProps {
+    config: any;
+}
+
+const FormBuilder: React.FC<FormBuilderProps> = ({ config }) => {
+    return (
+        <></>
```
Comment (IgorKravtsov) [inline src/components/FormBuilder/FormBuilder.tsx:9]: Nice try ^)

File: src/pages/Login/config.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  type: string,
+  validation: [
+    {
+      name: string,
+      validValue: number,
+    }
+  ]
+}
+
+const config = [
```
Comment (IgorKravtsov) [inline src/pages/Login/config.ts]: Add type for this array

File: src/pages/Login/config.ts [hunk 5]
```diff
@@ -0,0 +1,51 @@
+interface configProps{
```
Comment (IgorKravtsov) [inline src/pages/Login/config.ts]: move this interface to interfaces folder (path src/interfaces/config-props.ts)

Comment (Reinmir) [reply src/pages/Login/config.ts]: Created new file config-props.ts.

File: src/pages/Login/config.ts (hunk 4 above)
Comment (Reinmir) [reply src/pages/Login/config.ts]: Added type for this array.

File: src/components/App/App.tsx (hunk 2 above)
Comment (Reinmir) [reply src/components/App/App.tsx]: Moved wrapper Router to index.tsx.

File: package.json (hunk 1 above)
Comment (Reinmir) [reply package.json]: Changed the nessesary lib

**Review by IgorKravtsov — APPROVED** [review]

**Comment by IgorKravtsov** [comment]
Please merge

**Pull request merged**